docs/automated-testing/getting-started.md

```diff
@@ -0,0 +1,125 @@
+---
+sidebar_label: Getting Started
+---
+
+# Getting Started with Automated Testing
+
+This quick walk-through will show you how to integrate the Regression Games Automated Testing package into your game, 
+and how to use the built-in "Monkey Bot",
```

Comment:
I'm not sold on calling this a walkthrough or limiting it to the scope of the Monkey Testing bot. This page isn't in the tutorials section of the docs site, so I'd expect it to encompass every available feature of the Automated Testing package - for example, the manual recording option (which I don't see documented yet)

Reply:
Eh. Maybe that's fine for now until there are more bots/features to document

Reply:
> Our package's Monkey Bot will mimic real user behavior and record the state of your game as it performs its testing.

I like the rewrite but might adjust this sentence a bit. I don't want to mislead users into thinking the bot will behave like a user. It certainly doesn't, it's quite random and I think we should be clear about that for now. As we integrate LLMs, it can get smarter and we can update the docs.

Reply:
My suggestion:

> This quick walkthrough will teach you how integrate the Regression Games Automated Testing package into your game and leverage a built-in bot to perform Monkey Testing. Monkey Testing is a strategy where a user randomly interacts with UI elements to uncover potential edge-cases. The built-in "Monkey Bot" will allow you to do some basic UI hardening to ensure your game doesn't crash or behave unexpectedly when a user clicks on random UI elements.

Reply:
Good point! I like your rewrite.